modules/features/pages/plugins.adoc

```diff
@@ -1,3 +1,133 @@
+
+== Red Hat Supported Plugins
+
+Red Hat actively participates in the thriving Backstage community by developing and supporting a number of useful plugins that are pre-integrated into RHDH by default.
+Some of the most important ones are listed below. There are plans to develop more plugins that integrate with the Red Hat product ecosystem.
+
+=== Tekton Plugin
+
+The Tekton plugin enables you to visualize the results of CI/CD pipeline runs on your Kubernetes or OpenShift clusters.
+The plugin allows users to visually see high level status of all associated tasks in the pipeline for their applications.
+
+image::rhdh-tekton-plugin.png[title=RHDH Tekton Plugin,width=800px]
+
+This plugin is similar to the pipeline view in the **Developer Perspective** of the OpenShift Console, and will continue to be enhanced with more features.
+
+image::rhdh-tekton-viz.png[title=RHDH Tekton Pipeline Visualization,width=800px]
+
+=== ArgoCD Plugin
+
+This plugin allows you to visualize the ArgoCD sync status, deployment history, and health status for your GitOps enabled applications.
+
+image::rhdh-argocd-plugin.png[title=RHDH ArgoCD Plugin,width=800px]
+
+The ArgoCD plugin can show you the high-level status of a GitOps enabled application in your Catalog.
+
+image::rhdh-argocd-viz.png[title=RHDH ArgoCD Sync Status,width=800px]
+
+=== Quay.io Plugin
+
+This plugin displays information about container images of your applications in the Quay registry.
+
+image::rhdh-quay-plugin.png[title=RHDH Quay.io Plugin,width=800px]
+
+The plugin also shows any critical vulnerabilities that were discovered while scanning the container image.
+
+image::rhdh-quay-sec-scan.png[title=RHDH Quay.io Plugin Security Scanner,width=800px]
+
+=== Keycloak Plugin
+
+The Keycloak plugin allows RHDH to provide Single Sign-On (SSO) features for the developer portal. It maps the Keycloak user and group entities to corresponding users and groups (teams) in RHDH. Integrating with Keycloak enables the use of multiple authentication and providers, for example Social login (Google, Microsoft, Meta, Twitter/X and more), Active Directory Authentication and SAML 2.0 based identity providers, OAuth and OIDC compliant identity providers, and more.
+
+The Keycloak backend plugin has the following capabilities:
+
+* Synchronization of Keycloak users in a realm
+* Synchronization of Keycloak groups and their users in a realm
+
+image::rhdh-keycloak-plugin.png[title=RHDH Keycloak Plugin,width=800px]
+
+Users and Groups from the Keycloak domain are mapped to users and groups (teams) in RHDH.
+
+image::rhdh-keycloak-users.png[title=RHDH Keycloak Users and Groups,width=800px]
+
+=== Topology Plugin
+
+The Topology plugin enables you to visualize the workloads such as Deployment, Job, Daemonset, Statefulset, CronJob, and Pods for any service on your Kubernetes or OpenShift cluster.
+The plugin originated from the **Developer Perspective** in the OpenShift Console.
```

Comment:
More specifically, I assume you mean the OpenShift Topology View of a Project in the Developer Perspective?

Reply:
yes. re-worded